issuelinktype.go

```diff
@@ -96,16 +112,26 @@ func (s *IssueLinkTypeService) Update(linkType *IssueLinkType) (*IssueLinkType,
 	return &ret, resp, nil
 }
 
+// Update wraps UpdateWithContext using the background context.
+func (s *IssueLinkTypeService) Update(linkType *IssueLinkType) (*IssueLinkType, *Response, error) {
+	return s.UpdateWithContext(context.Background(), linkType)
+}
+
 // Delete deletes an issue link type based on provided ID.
 //
 // JIRA API docs: https://developer.atlassian.com/cloud/jira/platform/rest/v2/#api-rest-api-2-issueLinkType-issueLinkTypeId-delete
-func (s *IssueLinkTypeService) Delete(ID string) (*Response, error) {
+func (s *IssueLinkTypeService) linkType(ctx context.Context, ID string) (*Response, error) {
```

Comment:
This seem to by a typo.
Is this ... ?

```suggestion
func (s *IssueLinkTypeService) DeleteWithContext(ctx context.Context, ID string) (*Response, error) {
```